hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -931,6 +941,21 @@ public int getMetadataCleanerCommitsRetained() {
     return Integer.parseInt(props.getProperty(HoodieMetadataConfig.CLEANER_COMMITS_RETAINED_PROP));
   }
 
+  /**
+   * File system IO buffering configs.
+   */
+  public boolean isFileIOBufferingEnabled() {
+    return Boolean.parseBoolean(props.getProperty(FILE_IO_BUFFER_ENABLED, DEFAULT_FILE_IO_BUFFER_ENABLED));
+  }
+
+  public int getFileIOBufferiMinSize() {
```

Comment:
minor typo. "g" for buffering is missing

Reply:
Fixed